pages/datasets/_datasetId.vue

```diff
@@ -294,19 +294,24 @@ const getThumbnailData = async (datasetDoi, datasetId, datasetVersion, datasetFa
           if (speciesArray && speciesArray.length > 0)
             species = speciesArray[0].children[0].label.toLowerCase()
         }
-        let taxo = Uberons.species['rat']
 
-        if (species && species in Uberons.species)
-          taxo = Uberons.species[species]
+        // check if there is a flatmap for the given species
+        let taxo = species && species in Uberons.species ? Uberons.species[species] : undefined
```

Comment:
Is there a reason why this isn't `const taxo = species && species in Uberons.species ? Uberons.species[species] : Uberons.species['rat']`?